src/olympia/scanners/tests/test_tasks.py

```diff
@@ -62,6 +63,10 @@ def test_run_with_mocks(self, requests_mock):
         assert result.upload == upload
         assert result.scanner == self.FAKE_SCANNER
         assert result.results == scanner_data
+        scanner_name = dict(self.MOCK_SCANNERS).get(self.FAKE_SCANNER)
```

Comment:
I guess not a question specific to this patch, but if we have to do `dict()` everywhere `SCANNERS` is accessed, why not just define it as a dict in the first place?

Reply:
Can we pass a `dict` to a [Django's `choices` model attribute](https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.Field.choices)?

Reply:
No, you'd have to do dict.items() to get a list of tuples back.  (Which I realise is just the opposite way around)  You only have to do that in one place vs. converting to a dict in many places though.  A nit for any follow-up patches that touch the scanners/yara code I suppose.

Reply:
yep, I agree it sounds nicer.